ci/build-docs.sh

```diff
@@ -1,9 +1,9 @@
 #!/bin/bash
 
-tracked_markdown=$(git ls-tree -r HEAD --name-only docs | egrep '\.(markdown|md)$')
+git ls-tree -r HEAD --name-only docs | egrep '\.(markdown|md)$' > tracked_markdown
 
-gelatyx lua --file $tracked_markdown --language-config ci/stylua.toml
-gelatyx lua --file $tracked_markdown --language-config ci/stylua.toml --check || exit 1
+gelatyx --language lua --file-list tracked_markdown --language-config ci/stylua.toml
+gelatyx --language lua --file-list tracked_markdown --language-config ci/stylua.toml --check
```

Comment:
I think the exit is still needed here, as the script isn't running with `-e`:

```suggestion
gelatyx --language lua --file-list tracked_markdown --language-config ci/stylua.toml --check || exit 1
```